cristin-person/src/main/java/no/unit/nva/cristin/person/handler/DeletePersonEmploymentHandler.java

```diff
@@ -0,0 +1,72 @@
+package no.unit.nva.cristin.person.handler;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.cristin.common.client.CristinAuthenticator;
+import no.unit.nva.cristin.person.client.DeletePersonEmploymentClient;
+import no.unit.nva.utils.AccessUtils;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.RestRequestHandler;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+
+import java.net.HttpURLConnection;
+
+import static no.unit.nva.cristin.common.Utils.getValidEmploymentId;
+import static no.unit.nva.cristin.common.Utils.getValidPersonId;
+
+@SuppressWarnings({"unused","PMD.SingularField"})
+public class DeletePersonEmploymentHandler extends ApiGatewayHandler<Void, Void> {
+
+
+    private final transient DeletePersonEmploymentClient apiClient;
+
+    public DeletePersonEmploymentHandler() {
+        this(new DeletePersonEmploymentClient(CristinAuthenticator.getHttpClient()), new Environment());
+    }
+
+    public DeletePersonEmploymentHandler(DeletePersonEmploymentClient apiClient, Environment environment) {
+        super(Void.class, environment);
+        this.apiClient = apiClient;
+    }
+
+
+
+
+    /**
+     * Implements the main logic of the handler. Any exception thrown by this method will be handled by {@link
+     * RestRequestHandler#handleExpectedException} method.
+     *
+     * @param input       The input object to the method. Usually a deserialized json.
+     * @param requestInfo Request headers and path.
+     * @param context     the ApiGateway context.
+     * @return the Response body that is going to be serialized in json
+     * @throws ApiGatewayException all exceptions are caught by writeFailure and mapped to error codes through the
+     *                             method {@link RestRequestHandler#getFailureStatusCode}
+     */
+    @Override
+    protected Void processInput(Void input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        AccessUtils.validateIdentificationNumberAccess(requestInfo);
```

Comment:
that or `requesterIsUserAdministrator` ?

Reply:
````
private void validateHasAccessRights(RequestInfo requestInfo) throws ForbiddenException, UnauthorizedException {
    if (!AccessUtils.requesterIsUserAdministrator(requestInfo)) {
        throw new ForbiddenException();
    }
}
````

Reply:
Oh yes, requester must be user administrator